extension.js

```diff
@@ -0,0 +1,87 @@
+var successColor = '#5cb85c';
+var warningColor = '#f0ad4e';
+var errorColor = '#d9534f';
+var okLimit = 1;
+var warningLimit = 3;
+var pollInterval = 1;   //in minutes
+var issuesURL = 'https://api.github.com/orgs/renuo/issues';
+
+var pullRequestsURL = 'https://github.com/pulls/assigned';
+var optionsURL = 'chrome://extensions/?options=' + chrome.runtime.id;
+
+var currentURL = pullRequestsURL;
+
+setInterval(performCall, pollInterval * (60 * 1000));  //every five minutes
+init();
+
+function openCurrentURL() {
+    chrome.tabs.create({'url': currentURL}, function (tab) {
+        // Tab opened.
+    });
+}
+function init() {
+    currentURL = pullRequestsURL;
+    chrome.browserAction.onClicked.removeListener(openCurrentURL);
+    chrome.browserAction.onClicked.addListener(openCurrentURL);
```

Comment:
Maybe it would be better to just have one listener which checks if it should go to the `optionsUrl` or to the `pullRequestsUrl`.